ledger/ledger_test.go

```diff
@@ -364,7 +371,10 @@ func TestLedgerBlockHeaders(t *testing.T) {
 func TestLedgerSingleTx(t *testing.T) {
 	a := require.New(t)
 
-	genesisInitState, initSecrets := testGenerateInitState(t, protocol.ConsensusV7)
+	// V15 is the earliest protocol version in active use.
+	// The genesis for betanet and testnet is at V15
+	// The genesis for mainnet is at V17
+	genesisInitState, initSecrets := testGenerateInitState(t, protocol.ConsensusV15)
```

Comment:
that's good observation; V7 is where Adam stopped with the consensus parameter folding.